pkg/utils/try-close.go

```diff
@@ -0,0 +1,14 @@
+///////////////////////////////////////////////////////////////////////
```

Comment:
NIT: should use _ instead of - in file name (but TBH, `close.go` is a perfectly fine name for this as well :) )

Reply:
yep, `close.go` works